src/decorators/tooltip/tooltip.js

```diff
@@ -0,0 +1,73 @@
+/*jshint unused:false */
+/*jshint -W025 */
+
+function(node, options) {
+	var tooltip, handlers, eventName, enterSection, leaveSection,
+		config = {
+			tagElement: options.tagElement || 'span',
+			className: options.className || 'tooltip',
+			selectorName: options.selectorName || 'tooltip' + Math.floor((Math.random() * 1000) + 1),
```

Comment:
So, the probability of generating identical selectorNames (if not provided) is 1/1000? 0.1%?